releasenotes/notes/cws-7-49-5a11d31bba5f9051.yaml

```diff
@@ -0,0 +1,22 @@
+---
+features:
+  - |
+    Detect patterns in processes and files paths to improve
```

Comment:
Should we tag all those entries with `CWS:` ? Otherwise this might end up in the middle of an incomprehensible mess of other entries no ?

Reply:
Good point. I used [CWS], is there any convention for this ?

Reply:
I'm not sure, but I think `[CWS] is fine`